understanding/22/focus-appearance.html

```diff
@@ -162,45 +162,45 @@ <h4>Other indicator shapes</h4>
                     difference between the areas of:</p>
 
                 <ul>
-                    <li>A 92px by 32px rectangle (1px larger on all sides), and</li>
-                    <li>A 88px by 28px rectangle (1px smaller on all sides)</li>
+                    <li>A 94px by 34px rectangle (2px larger on all sides), and</li>
+                    <li>A 90px by 30px rectangle (the area of the control itself)</li>
                 </ul>
 
-                <p>This results in a minimum area of (92px * 32px) - (88px * 28px) = 480px<sup>2</sup>.</p>
+                <p>This results in a minimum area of <code>(94px * 34px) - (90px * 30px) = 496px<sup>2</sup></code>.</p>
 
                 <p>Some general formulas for 2 CSS pixel thick perimeters of common shapes are:</p>
 
                 <dl>
                     <dt>Rectangle with width <em>w</em> and height <em>h</em></dt>
-                    <dd>4<em>h</em> + 4<em>w</em></dd>
+                    <dd><code>((<em>w</em> + 4) * (<em>h</em> + 4)) - (<em>w</em> * <em>h</em>)</code></dd>
                     <dt>Circle with radius <em>r</em></dt>
-                    <dd>4𝜋<em>r</em></dd>
+                    <dd><code>𝜋 * ((<em>r</em>+2)<sup>2</sup> - <em>r</em><sup>2</sup>)</code></dd>
                     <dt>Rounded rectangle with width <em>w</em>, height <em>h</em>, and border radius <em>r</em></dt>
-                    <dd>4<em>h</em> + 4<em>w</em> - (16 - 4𝜋)<em>r</em></dd>
+                    <dd><code>4 * (<em>w</em> + <em>h</em> - 4 * <em>r</em>) + 𝜋 * ((<em>r</em>+2)<sup>2</sup> - <em>r</em><sup>2</sup>)</code></dd>
                 </dl>
 
                 <p class="note">If you need to use complex mathematics to work out if a focus indicator is large enough,
                     it is probably a sign that you should use a larger indicator instead. The bigger the visible change
                     when an item receives focus, the easier it is for someone to see.</p>
 
                 <p>The following 2 examples use a 90px wide by 30px tall button, with a minimum area requirement of
-                    480px<sup>2</sup>:</p>
+                    496px<sup>2</sup>:</p>
 
                 <figure id="focus-indicator-custom-shapes-inset">
 
                     <img src="img/focus-indicator-custom-shapes-inset.png"
                         alt="Three 90 by 30 pixel buttons. The middle button has a focus indicator which is a 3px thick outline inset by 3px inside the button" />
-                    <figcaption>Passes: the inner outline is inset slightly from the outer edge of the component, but
-                        compensates for this by being 3px thick. It has an area of 612px<sup>2</sup>, which exceeds the
-                        480px<sup>2</sup> minimum.</figcaption>
+                    <figcaption>Passes: the inner outline is inset by 3px from the outer edge of the component, but
```

Comment:
I'm fine with clarifying "slightly" to "by 3px", but I'm confused about the switch from 612 to 606, I still get 612 when I recalculate. The outer edge of the indicator forms a rectangle 3px inset on all sides from 90x30 (ie, a 84x24 rectangle), and the inner edge of the indicator is inset 3px further from that (ie, a 78x18 rectangle), so I think the indicator area is $(84)(24)-(78)(18)=612$, no?